.tekton/multi-arch-build-pipeline.yaml

```
@@ -0,0 +1,622 @@
+    - name: IMAGE_URL
+      value: $(tasks.build-image-index.results.IMAGE_URL)
+    taskRef:
+      params:
+      - name: name
+        value: show-sbom
+      - name: bundle
+        value: quay.io/konflux-ci/tekton-catalog/task-show-sbom:0.1@sha256:beb0616db051952b4b861dd8c3e00fa1c0eccbd926feddf71194d3bb3ace9ce7
+      - name: kind
+        value: task
+      resolver: bundles
+    params:
```

YAMLlint (1.37.1)

[error] duplication of key "params" in mapping

(key-duplicates)